api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -478,9 +478,20 @@ private void handleWorkspaceLimit(long userId) {
   @Override
   public void updateWorkspaceBillingAccount(DbWorkspace workspace, String newBillingAccountName) {
     if (newBillingAccountName.equals(workspace.getBillingAccountName())) {
+      log.info(
+          String.format(
+              "Billing account for workspace %s is already set to %s",
+              workspace.getWorkspaceNamespace(), newBillingAccountName));
```

Comment:
if bug is here, then billing account name and billing status should be updated at the same time. But for our case, looks account_name is not changing?

Reply:
that's correct, however I wanted to have a better insight into what happens in this method generally even if it gets called with the same billing account name. Just to rule out any potential issues.